### PR TITLE
PT-BR translation

### PR DESCRIPTION
**Pull request by rickal**

### CODE REVIEW

File: translations/pt-BR.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      faq: "FAQ"
+    layout:
+      bitcoin_cash: "Bitcoin Cash"
+      lead: "Moeda eletrônica de pessoa para pessoa"
+      loading: "Carregando..."
+    features:
+      - title: "Rapidez"
+        description: "Transacione em segundos. Obtenha confirmação em minutos."
+      - title: "Confiabilidade"
+        description: "Uma reder que funciona sem congestionamento."
```
Comment (felipeagc) [inline translations/pt-BR.yml]: Found a typo: "reder"

Comment (rickal) [reply translations/pt-BR.yml]: Yes, thank you Felipe!

**Comment by m4ktub** [comment]
Just to note that this PR replaces `en.yml` by `pt-BR.yml` instead of just adding `pt-BR.yml`. As such, it won't be merged.

**Comment by rickal** [comment]
I'll create another one.

**Pull request closed without merging**